bravado/mapping/swagger20_validator.py

```diff
@@ -0,0 +1,39 @@
+from jsonschema import validators, _validators
+from jsonschema.exceptions import ValidationError
+from jsonschema.validators import Draft4Validator
+
+
+def ignore(_validator, *args):
+    """A validator which performs no validation. Used to `ignore` some schema
+    fields during validation.
+    """
+    return
+
+
+def required_validator(validator, req, instance, schema):
+    """Swagger 2.0 expects `required` to be a bool in the Parameter object,
+    but a list of properties everywhere else.
+    """
+    # TODO: Find a better way to identify if it is a Parameter schema
+    if schema.get('in'):
```

Comment:
This would probably benefit from being moved to `bravado.mapping.spec.is_param_spec(..)` where similar friends live.